src/CoreMedia/CMSync.cs

```diff
@@ -411,20 +468,30 @@ public CMTimebaseError SetTimerToFireImmediately (NSTimer timer)
 
 #if !NET
 		[TV (13,0), Mac (10,15), iOS (13,0)]
```

Comment:
Should this line be removed as well?

Reply:
What? The deprecation is needed for classic. The unsupported is needed as well, else it will use the default, since the minimum versions are 10 for iOS/tvOS, 10.4 for mac OS x and no watch. Without the Unsupported it will be assumed that they are present in those platforms. If anything we could remove the os version number.